device/bgp/bgpsrteadvanced.yaml

```diff
@@ -0,0 +1,25 @@
+components:
+  schemas:
+    BgpSrTe.Advanced:
```

Comment:
The description says Route Range. Shall we have RR in the name to indicate that i.e. "BgpSrTe.Advanced" becomes "BgpSrTeRR.Advanced"?